src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -654,6 +654,42 @@
     <Field Name="Binder" Type="Binder" Null="disallow"/>
     <Field Name="MemberAccess" Type="BoundLateMemberAccess" Null="disallow"/>
   </Node>
+
+  <!-- 
+  Tuple literals can exist in two forms - literal and converted literal.
+  This is the base node for both forms.
+  -->
+  <AbstractNode Name="BoundTupleExpression" Base="BoundExpression">
+    <Field Name="Arguments" Type="ImmutableArray(Of BoundExpression)"/>
+  </AbstractNode>
+
+  <!-- 
+  Tuple literals can convert to the target type.
+  Once converted to a target type, they cannot be target-typed again. 
+  The tuple literal is one which has not been converted to a target type.
+  -->
+  <Node Name="BoundTupleLiteral" Base="BoundTupleExpression">
+    <!-- 
+    It is possible for a tuple to not have a type in a literal form
+    Ex: 
+     (a: 1, b: (c: 1, d: null))  does not have a natural type, because "null" does not have one
```

Comment:
'Nothing' in VB actually has a default type - it is Object, but it gets that type only after reclassification and if happens to be not target typed, so it will work here as expected.
I think we would need to rationalize what 'natural' means in VB when conversions/classifications are implemented. Or we may remove the notion of the natural type altogether.

---
In reply to: [71451171](https://github.com/dotnet/roslyn/pull/12596#discussion_r71451171) [](ancestors = 71451171,71443934)